ckanext/datatablesview/plugin.py

```diff
@@ -96,3 +98,10 @@ def info(self) -> dict[str, Any]:
                 u'filterable': [default(True), boolean_validator],
             }
         }
+
+    # ITemplateHelpers
+
+    def get_helpers(self) -> dict[str, Callable[..., object]]:
+        return {'get_null_label': lambda: toolkit._(self.null_label)
```

Comment:
how about `datatablesview_null_label` or similar? template helpers share a common namespace.

Also using a lambda here is no shorter than defining a normal function, so I'd use a normal function instead.

Reply:
renamed to `datatablesview_null_label` and added helpers module instead of lambda function.